mindone/diffusers/pipelines/stable_diffusion_xl/__init__.py

```diff
@@ -1,2 +1,4 @@
 from .pipeline_output import StableDiffusionXLPipelineOutput
+from .pipeline_stable_diffusion_inpainting import StableDiffusionXLInpaintPipeline
 from .pipeline_stable_diffusion_xl import StableDiffusionXLPipeline
+from .pipeline_stable_diffusion_xl_img2img import StableDiffusionXLImg2ImgPipeline
```

Comment:
BTW, i forgot to add `_import_structure` for lazy importing. Please implement it~

Reply:
Implemented the lazy importing structure following the example in stable_diffusion folder